nbs/models.tft.ipynb

```diff
@@ -15,20 +15,14 @@
    "source": [
```

Comment:
<p>I suggest "Temporal Fusion Transformer (TFT) methods" to make search easier. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
Changed model title to TFT for homogeneity across models, at the title level we will keep acronyms.

Reply:
<p>Great! Lets try to include the full titles in the text then to make it easier for search engines to find. </p>